app/policies/roles/institution/health_organization.json.erb

```diff
@@ -10,20 +10,14 @@
             "institution:read",
             "institution:update",
             "institution:delete",
-            "institution:registerDevice",
-            "institution:createSite",
-            "institution:createRole"
+            "institution:createRole",
+            "institution:readUsers"
           ],
           "delegable": true
         },
         {
-          "resource": [
-            "site?institution=<%= institution_id %>",
-            "device?institution=<%= institution_id %>",
-            "testResult?institution=<%= institution_id %>",
-            "encounter?institution=<%= institution_id %>"
-          ],
-          "action": "*",
+          "resource": "testResult?institution=<%= institution_id %>",
+          "action": "testResult:medicalDashboard",
```

Comment:
The action checked for which test results are to be displayed in the dashboard is `testResult:query`; `testResult:medicalDashboard` is only checked to determine whether the user has access to the dashboard itself or not.